src/embeddings.rs

```diff
@@ -289,6 +289,10 @@ where
             VocabWrap::ExplicitSubwordVocab(_) => {
                 Err(Error::conversion_error("ExplicitVocab", "ExplicitVocab"))
             }
+            VocabWrap::FloretSubwordVocab(_) => Err(Error::conversion_error(
+                "FloretSubwordVocab",
+                "ExplicitVocab",
+            )),
```

Comment:
Not part of the changed lines so I'm commenting down here: The documentation is out of sync with this change since the floret vocab doesn't support (in the current state) conversion to an explicit subword vocab

Reply:
Fixed.